### PR TITLE
yamllint formatting cleanup for ocp-workload-edge-deployments

### PR DESCRIPTION
**Pull request by ritesh97-rh**
##### SUMMARY
For the specified roles:
- Create a yamllint file to enforce formatting rules
- Make changes to existing code base to adhere to formatting requirements
- yamllint errors will be displayed during formatting checks in future development

##### ISSUE TYPE
- Code Cleanup Request

##### COMPONENT NAME
ocp-workload-edge-deployments

### CODE REVIEW

File: ansible/roles/ocp-workload-edge-deployments/files/project-iot-development.yaml [hunk 1]
```diff
@@ -598,8 +599,10 @@ objects:
                           script {
                               openshift.withCluster() {
                                   openshift.withProject() {
-                                      // if everything else succeeded, tag the ${templateName}:latest image as ${testingProject}/${templateName}:latest
-                                      openshift.tag("${openshift.project()}/${templateName}:latest", "${testingProject}/${templateName}:latest")
+                                      // if everything else succeeded, tag the ${templateName}:latest image
+                                      // as ${testingProject}/${templateName}:latest
+                                      openshift.tag("${openshift.project()}/${templateName}:latest",
+                                      "${testingProject}/${templateName}:latest")
```
Comment (stencell) [inline ansible/roles/ocp-workload-edge-deployments/files/project-iot-development.yaml:605]: This is how the jenkinsfile is written. I'm not certain you can just move this to another line. It is not a ansible or yaml multiline string.

This must be tested.

File: ansible/roles/ocp-workload-edge-deployments/files/project-iot-development.yaml [hunk 2]
```diff
@@ -742,8 +745,10 @@ objects:
...
```
Comment (stencell) [inline ansible/roles/ocp-workload-edge-deployments/files/project-iot-development.yaml:751]: This must be tested since it is code in the jenkinsfile.

File: ansible/roles/ocp-workload-edge-deployments/files/project-iot-development.yaml [hunk 3]
```diff
@@ -886,8 +891,10 @@ objects:
...
```
Comment (stencell) [inline ansible/roles/ocp-workload-edge-deployments/files/project-iot-development.yaml:897]: same here

**Review by stencell — CHANGES_REQUESTED** [review]
comments to be fixed as noted.

File: ansible/roles/ocp-workload-edge-deployments/files/project-iot-development.yaml (hunk 1 above)
Comment (ritesh97-rh) [reply ansible/roles/ocp-workload-edge-deployments/files/project-iot-development.yaml:605]: Ah, you may be right. It didn't look like a yaml block, so I assumed that `//` was the comment char.

Comment (stencell) [reply ansible/roles/ocp-workload-edge-deployments/files/project-iot-development.yaml:605]: Lines 602 and 603 are comments...but 604 & 605 are the code and I think that is what needs to be reverted since they are writing this jenkinsfile inline. Same with other comments.

Comment (ritesh97-rh) [reply ansible/roles/ocp-workload-edge-deployments/files/project-iot-development.yaml:605]: Doesn't groovy ignore line breaks? The code might look slightly different (have an extra line break), but would it not run the same?

Comment (stencell) [reply ansible/roles/ocp-workload-edge-deployments/files/project-iot-development.yaml:605]: I am really not sure...hence my hesitation without either someone who knows for sure or with testing.

File: ansible/roles/ocp-workload-edge-deployments/files/deploy-containers-apb/playbooks/provision.yml [hunk 4]
```diff
@@ -41,47 +42,91 @@
...
+      docker run -d
+      --name software-sensor
+      --link amq63:amq63
+      --restart always
+      -e JAVA_OPTIONS='
+      -DhighWater=800 -DlowWater=200
+      -DbrokerUID=admin -DbrokerPassword=change12_me
+      -DreceiverURL=amq63 -DdeviceType=temperature
+      -DdeviceID=4711 -DinitialValue=70
+      -Dcount=1000 -DwaitTime=1'
```
Comment (jkupferer) [inline ansible/roles/ocp-workload-edge-deployments/files/deploy-containers-apb/playbooks/provision.yml:130]: Is is still rather hard to follow...

**Review by jkupferer — CHANGES_REQUESTED** [review]
